### PR TITLE
Docs: Add plugin-nornir and sort links alphabetically

### PR DESCRIPTION
**Pull request by cmsirbu**
# Closes: #DNE
# What's Changed

Updated Apps section in docs with the shiny new Nornir plugin docs site and reshuffled the links pseudo alphabetically in the table.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -199,8 +199,10 @@ nav:
   - Nautobot Apps:
       - Overview: "apps/index.md"
       - Network to Code App Docs:
+          - Nautobot ChatOps: "https://docs.nautobot.com/projects/chatops/en/latest/"
+          - Nautobot Firewall Models: "https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/"
+          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
```
Comment (glennmatthews) [inline mkdocs.yml:204]: Should this be `Nautobot Nornir Plugin` instead? `Nautobot Plugin Nornir` reads awkwardly to me. I know we need to distinguish between `nautobot-plugin-nornir` and `nornir-nautobot` though.

Comment (itdependsnetworks) [reply mkdocs.yml:204]: Unfortunately, this is correct, we documented the faq (the second one here) https://docs.nautobot.com/projects/plugin-nornir/en/latest/user/app_faq/ to explain why.

Comment (cmsirbu) [reply mkdocs.yml:204]: @glennmatthews I just take whatever the plugin calls itself officially :) I agree that the naming in this case is a bit awkward, but I don't think there's much to be done about it right now.

**Review by glennmatthews — APPROVED** [review]

**Comment by glennmatthews** [comment]
Please add a `changes/2512.added` changelog fragment file, probably with contents like:

```
Added link to `nautobot-plugin-nornir` in the documentation.
```

**Comment by cmsirbu** [comment]
> Please add a `changes/2512.added` changelog fragment file, probably with contents like:
> 
> ```
> Added link to `nautobot-plugin-nornir` in the documentation.
> ```

Done!

**Review by bryanculver — APPROVED** [review]

**Pull request merged**